src/libraries/Microsoft.PowerFx.Core/Microsoft.PowerFx.Core.csproj

```diff
@@ -29,10 +29,10 @@
 	</ItemGroup>
 
 	<ItemGroup>
-		<EmbeddedResource Include="$(SolutionDir)\strings\**\PowerFxResources.resw" Condition="'$(Configuration)' == 'Debug'">
+		<EmbeddedResource Include="..\..\strings\**\PowerFxResources.resw" Condition="'$(Configuration)' == 'Debug'">
```

Comment:
Why this change?

Reply:
it has no real impact, just curious.

Reply:
When I'm working on the code to convert from Adaptive to Pfx I add the Pfx projects to the Conversion solution and then edit/build/debug across both code bases which allows me to work on both at the same time. But having $(SolutionDir)\ as the path for the string resources doesn't work if you add the Pfx projects to a different solution and the solution isn't in the same folder tree. Very esoteric, but as you said, it really doesn't change anything on the Pfx side in that today both are equivalent anyway.